nixos/modules/services/networking/rpki-client.nix

```diff
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.rpki-client;
+  tals = [
+    "${cfg.package}/etc/rpki/afrinic.tal" "${cfg.package}/etc/rpki/apnic.tal"
+    "${cfg.package}/etc/rpki/lacnic.tal" "${cfg.package}/etc/rpki/ripe.tal" ]
+    ++ optional cfg.accept-arin-rpa "${pkgs.rpki-arin-tal}/arin.tal"
+    ++ cfg.additional-tals;
+in {
+  options = {
+    services.rpki-client = {
+      enable = mkEnableOption (lib.mdDoc "Enable OpenBSD rpki-client");
+
+      package = mkOption {
+        description = (lib.mdDoc "OpenBSD rpki-client");
+        type = types.package;
+        default = pkgs.rpki-client;
+        defaultText = literalExpression "pkgs.rpki-client";
+      };
+
+      accept-arin-rpa = mkOption {
+        description = (lib.mdDoc "Accept the ARIN RPA and download the ARIN TAL");
+	type = types.bool;
+        default = false;
+      };
+      
+      additional-tals = mkOption {
+        description = (lib.mdDoc "list of additional tals");
+	type = types.listOf types.str;
+        default = [ ];
+      };
+
+      timer.enable = mkOption {
+        description = (lib.mdDoc "Enable systemd timer to generate rpki config");
+	type = types.bool;
+        default = true;
+      };
+
+      timer.interval = mkOption {
+        default = "1h";
+        type = types.str;
+        description = lib.mdDoc ''
+          The interval at which to run rpki-client and update the lists.
+          See {command}`man 7 systemd.time` for the format.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    environment.systemPackages = [ cfg.package ];
```

Comment:
Is this used as a cli? Otherwise it should only be added to the service.

Reply:
yes, it is also used as cli tool